health/state.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package health
+
+import (
+	"time"
+)
+
+// State is a representation of the health status of the backend, which is
+// provided to the client UI via LocalAPI. It is used to simplify the data structure
```

Comment:
I'd remove the "It is" sentence. It invites more questions than it answers. (simplifies compared to what?)